lib/wanda/messaging/adapters/amqp.ex

```diff
@@ -8,7 +8,13 @@ defmodule Wanda.Messaging.Adapters.AMQP do
   alias Wanda.Messaging.Adapters.AMQP.Publisher
 
   @impl true
+
+  # FIXME: fix Trento.Contracts.to_event/2
+  @dialyzer {:nowarn_function, publish: 2}
+
   def publish(routing_key, message) do
-    Publisher.publish_message(message, routing_key)
+    message
+    |> Trento.Contracts.to_event(source: "wanda")
```

Comment:
Maybe the source could be `github.com/trento-project/wanda`

Reply:
good one, we could take this as inspiration: https://github.com/googleapis/google-cloudevents/blob/main/examples/structured/pubsub/MessagePublishedData.json